src/components/Cart/index.js

```diff
@@ -0,0 +1,76 @@
+import { useData } from "../../Context/stateManage/state-context";
+import "./cart.css";
+import CartProduct from "./CartCard/CartProduct";
+
+const Cart = () => {
+  const {
+    state: { cartItems },
+  } = useData();
+
+  return (
+    <main className='cart-wrapper'>
+      <div className='cart-head flex jc-center'>
+        <h1>My Cart {cartItems.length}</h1>
+      </div>
+      <section className='pd-container-main flex flex-wrap flex-gap jc-center'>
+        <div className='pd-wrapper flex jc-center flex-col flex-gap flex-wrap'>
+          {cartItems.length > 0 ? (
+            cartItems.map((item) => (
+              <CartProduct
+                key={item._id}
+                id={item._id}
```

Comment:
Pass the whole props instead and destructure on cart products

Reply:
Ok  i will do this .